transformers/shared/constants/method.go

```diff
@@ -173,6 +173,9 @@ func vatSlipMethod() string { return getSolidityFunctionSignature(VatABI(), "sli
 func vatSuckMethod() string { return getSolidityFunctionSignature(VatABI(), "suck") }
 func vowFessMethod() string { return getSolidityFunctionSignature(VowABI(), "fess") }
 func vowFileMethod() string { return getSolidityFunctionSignature(VowABI(), "file") }
```

Comment:
A little surprised this doesn't need to use `getOverloadedFunctionSignature` - maybe worth using anyway to be explicit about the distinction?

Reply:
🤔  Yeah, good catch. I wonder if we just lucked out before, and the file signature we were after was returned first from the abi. Updating it to use `getOverloadedFunctionSignature` makes sense to me though.